vector/src/main/java/im/vector/app/features/VectorFeatures.kt

```diff
@@ -40,5 +41,6 @@ class DefaultVectorFeatures : VectorFeatures {
     override fun isOnboardingSplashCarouselEnabled() = true
     override fun isOnboardingUseCaseEnabled() = true
     override fun isOnboardingPersonalizeEnabled() = false
+    override fun isOnboardingCombinedChooseServerEnabled() = false
```

Comment:
We could also rename the feature flag as `isOnboardingCombinedRegisterEnabled()`. What do you think?

Reply:
agreed :+1:  https://github.com/vector-im/element-android/pull/5648/commits/aac206f1ac46aa22239950ae5ca2d34162b346e3

Reply:
Thanks! I see a last thing to rename, the boolean inside the `Factory`: 
```
createBooleanFeature(
    label = "FTUE Combined register",
    key = DebugFeatureKeys.onboardingCombinedRegister,
    factory = VectorFeatures::isOnboardingCombinedRegisterEnabled
)
```

Reply:
I should have done a search replace :sweat_smile: amended the commit above ^^^ https://github.com/vector-im/element-android/pull/5648/commits/30635af0dbcec3650c72b724f10331319355dd2d

Reply:
Was it intented to not rename the key `DebugFeatureKeys.onboardingCombinedChooseServer`? Sorry to annoy you with namings...

Reply:
this is my bad! third time lucky :crossed_fingers:  https://github.com/vector-im/element-android/pull/5648/commits/5120e7a8c68f5d2e8d36f6186f8543b6b5eed89d